js/pricing.js

```diff
@@ -9,17 +12,44 @@ $(
     function () {
         const $confirmPersonalInformation = $('#confirm-personal-information');
         const $confirmDevelopmentAgreement = $('#confirm-development-agreement');
-        const $disabledButton = $('#order-now-btn');
+        const $orderButton = $('#order-now-btn');
+        const $redirect = $('#redirect');
+        const $loader = $('.redirect-loader', '#redirect');
+        const $errorContainer = $('.redirect-error', '#redirect');
+        const $tryAgain = $('#try-again');
+
+        const orderUrl = "{{site.data.backend_links.orderUrl}}";
+        const prodApiUrl = "{{site.data.backend_links.prodApiUrl}}";
+        const devApiUrl = "{{site.data.backend_links.devApiUrl}}";
+        const registerTransactionPath = "{{site.data.backend_links.registerTransactionPath}}";
+        // `window.mode` we setup in the `getting-help/service-section.html` file by Jekyll.
+        const apiUrl = window.mode === "development" ? devApiUrl : prodApiUrl;
+
 
         $confirmPersonalInformation.change(function () {
-            confirmAgreement(this, $disabledButton);
+            confirmAgreement($orderButton);
         });
 
         $confirmDevelopmentAgreement.change(function () {
-            confirmAgreement(this, $disabledButton);
+            confirmAgreement($orderButton);
+        });
+
+        $orderButton.click(function() {
+            submitOrder();
+        });
+
+        $tryAgain.click(function(event) {
+            event.preventDefault();
+            hideRedirect ();
+            submitOrder();
         });
 
-        function confirmAgreement(element, disabledElement) {
+        /**
+         * Disables and enables order button.
+         *
+         * @param {Object} disabledElement - disabled/enabled element
```

Comment:
Is it **just** an object? Or maybe a jQuery element? Or DOM element? Please consider having some dedicate type here instead of `Object`.